FileIO/Legacy/MeshIO.cpp

```diff
@@ -156,66 +175,66 @@ MeshLib::Element* MeshIO::readElement(const std::string& line,
 	{
 	case MeshElemType::LINE: {
 		for (int i = 0; i < 2; ++i)
-			ss >> idx[i];
+			in >> idx[i];
```

Comment:
for this and all the other element types: what happens if you run out of values to read? (e.g. you need to read two node ids for a line but what happens if the stream ends after the first value?)